src/main/java/org/assertj/core/internal/RealNumbers.java

```diff
@@ -93,5 +100,13 @@ public void assertIsNotInfinite(AssertionInfo info, NUMBER actual) {
     throw failures.failure(info, shouldNotBeInfinite(actual));
   }
 
+  public boolean isNanOrInfinite(NUMBER value){
+    return isNaN(value) || isInfinite(value);
+  }
```

Comment:
write a parameterized test for this method

Reply:
I would like to provide a javadoc for it. Is it a good idea?